README.md

```diff
@@ -1,35 +1,259 @@
-Ansible Role [![Build Status](https://github.com/onaio/ansible-role/workflows/CI/badge.svg)](https://github.com/onaio/ansible-role/actions?query=workflow%3ACI)
-=========
+# Ansible Database Migrations [![Build Status](https://github.com/onaio/ansible-database-migrations/workflows/CI/badge.svg)](https://github.com/onaio/ansible-database-migrations/actions?query=workflow%3ACI)
 
-A brief description of the role goes here.
+Use this role to set up a database using [sqitch](https://sqitch.org/) migrations.
 
-Requirements
-------------
+The database setup process is in three stages:
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+1. Set up
+   - One or more repositories that contain database migrations are cloned.  Both private and public git repositories can be cloned.  Private repositories require that you have the relevant ssh key added to ssh-agent.
+   - Create a [pgpass](https://www.postgresql.org/docs/9.3/libpq-pgpass.html) file to make authentication against the DB easy and standard throughout the role
+   - Execute any SQL queries that need to be run to set up the database.  These may be things such as creating databases and database users, creating postgres extensions etc.
+2. Sqitch migrations - the sqitch migrations are deployed and verified.  The role is able to support running multiple sqitch migration plans, for different databases.
+3. Set up periodic jobs - Ensure any SQL queries that need to be run periodically are set up a cron jobs.  These may be queries such as a daily data dump, refreshing materialized views, etc.
 
-Role Variables
---------------
+**Note**:
 
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+- At this moment, only PostgreSQL is supported.
+- This role does not concern itself with installing database systems.
 
-Dependencies
-------------
+## Role Variables
 
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
+Some of the more important variables are briefly described below.  You can see all variables by looking at the [defaults/main.yml](defaults/main.yml) file.
 
-Example Playbook
-----------------
+### database_migrations_system_user
 
-Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+This is the user which is used while running the role.  The default is "nomad" - you know, because they migrate :)
```

Comment:
I wonder, should we name the repository and role `ansible-nomad`? Or `ansible-sqitch-migrations`?

Reply:
I'll try `ansible-sqitch-migrations` :) - ansible-nomad is tempting but I think the generally accepted way to name ansible roles is in a way that describes what they do

Reply:
Done